src/Components/Activities/index.jsx

```diff
@@ -48,7 +50,7 @@ const Activities = () => {
       setBodyModal(data.message);
       setTypeStyle('success');
       setIsDelete(false);
-      setReload(true);
+      refreshTable();
```

Comment:
If you use the filter in reducer Delete you don't need to call refreshTable again.

Reply:
I already tried it, however if I take it out it does not update the list anyway

Reply:
resolved!